#!/usr/bin/env python

PACKAGE = "aiimooc_syz4"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sensor_height", double_t, 0, "height of lidar", 1.2, 1.0, 2.5)
gen.add("num_iter", int_t, 0, "number of iter", 6, 3, 10)
gen.add("num_lpr", int_t, 0, "number of intial seeds points", 20, 10, 200)
gen.add("th_seeds", double_t, 0, "threshold of get seeds", 0.2, 0, 0.5)
gen.add("th_dist",double_t,0,"threshold of get plane",0.3,0,0.5)

exit(gen.generate(PACKAGE, "lidar_pcl", "FitPlane"))
